stockroom/storages/model.py

```diff
@@ -0,0 +1,120 @@
+import warnings
+
+import numpy as np
+
+from .. import parser
+from ..repository import StockRepository
+from ..utils import LazyLoader
+
+torch = LazyLoader('torch', globals(), 'torch')
+tf = LazyLoader('tf', globals(), 'tensorflow')
+
+
+class ModelStore:
+    """
+    ModelStore class utilizes hangar arraysets to store the pieces of a model and
+    use hangar metadata to store the information required to collate it back to
+    a model. It right now supports ``keras.Model`` and ``torch.nn.Module`` models.
+    ModelStore instance, on :meth:`storages.ModelStore.save` creates few arraysets
+    (one arrayset for each data type) to store the weights and create one arrayset
+    specifically to store the shape of each layer. This shape arrayset is needed
+    because the weights of each layer would be flattened before saving. This is
+    essential since handling variable shapes and variable ranks are more complex
+    than flattening and reshaping-back the weights.
+    """
+
+    def __init__(self, write):
+        self.co = None
```

Comment:
Correct me if i'm wrong, but this instance attribute isn't actually used anywhere?

Reply:
Good catch. It's a residue from my old approach